<launch>
  <arg name="namespace" default="uwb_simulation_node" />
  <arg name="label_name" default="black_block2" />
  <arg name="anchor_model" default="black_block1" />
  <arg name="anchor_id" default="1" />

  <node pkg="uwb_simulation"
        exec="uwb_simulation_tag_to_anchor_node"
        namespace="$(var namespace)"
        output="screen">
    <param name="label_name" value="$(var label_name)" />
    <param name="anchor_model" value="$(var anchor_model)" />
    <param name="anchor_id" value="$(var anchor_id)" />
  </node>
</launch>